todo-list/src/TodoItem.js

```diff
@@ -1,10 +1,16 @@
 import React, { useState } from "react";
 import "./styles.css";
+import { COLOURS } from "./colours";
 
-function TodoItem({ index }) {
+function TodoItem({ index, isDarkMode }) {
```

Comment:
Another way to do this is by using props. It is way easier to pass parameters into components. Check out how I used it in the code I pushed!

Reply:
Fixed! Just a note, what I did was using props too. 

```
const TodoItem: React.FC<TodoItemProps> = ({ index, isDarkMode }) => {
  // ...
};
```

is equivalent to this

```
const TodoItem: React.FC<TodoItemProps> = (props) => {
  const { index, isDarkMode } = props;
  // ...
};

```

Just in the first one you don't need to use `props` when de-structuring in the `()`. But the second way is better once are lots of props and is a bit easier to understand what is going on.